.github/workflows/drivers_ci.yml

```diff
@@ -57,6 +57,15 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Install kernel headers (actuated)
```

Comment:
Duplicated (was already present below)?

Reply:
See starting from https://github.com/falcosecurity/libs/pull/1778/files#diff-d6454f674cf2f44e69373bc81e9062fa626ce079f23cb6a706f3b893fad6721bR82

Reply:
I saw that after 😓